src/Nethermind.Crypto.SecP256r1/Secp256r1.cs

```diff
@@ -33,42 +103,27 @@ public static partial class Secp256r1
     /// </returns>
     public static unsafe bool VerifySignature(in ReadOnlyMemory<byte> input)
     {
-        using MemoryHandle pin = input.Pin();
-        return VerifyBytes((byte*) pin.Pointer, input.Length) != 0;
-    }
-
-    private static void SetLibraryFallbackResolver()
-    {
-        Assembly assembly = typeof(Secp256r1).Assembly;
+        if (input.Length != 160)
+            return false;
 
-        AssemblyLoadContext.GetLoadContext(assembly)!.ResolvingUnmanagedDll += (context, name) =>
+        nint key = 0;
+        try
         {
-            if (context != assembly || !LibraryName.Equals(name, StringComparison.Ordinal))
-                return nint.Zero;
+            Span<byte> buffer = stackalloc byte[2 + 2 * (2 + 32 + 1)]; // Max possible size when DER-encoded
+            ReadOnlySpan<byte> signature = EncodeSignature(input.Span[32..64], input.Span[64..96], buffer);
```

Comment:
Looks like a split of a span. Again, I'd extract a meaningful constants, especially if 64 from the first is 64 from the second.